### PR TITLE
Featrue/introduce halls

### PR DESCRIPTION
**Pull request by MonikaLukowska**
This PR introduces the Hall model with its validation and rspec tests. 
I decided to skip controller and views as in my opinion listing halls or adding/updating them manually is not crucial at this stage of the project (can be implemented in future PR). Instead I added seeds so the db can be populated with halls according to the project specification.

### CODE REVIEW

**Review by Sonickers — APPROVED** [review]
LGTM 👍🏻

File: app/models/hall.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Hall < ApplicationRecord
+  validates :number, :capacity, presence: true
```
Comment (tobiaszwaszak) [inline app/models/hall.rb]: are you sure it should be the number? why not name? what if we want to name halls: red, white, or blue?

File: db/migrate/20220912190640_add_index_to_halls_number.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddIndexToHallsNumber < ActiveRecord::Migration[7.0]
+  def change
+    add_index :halls, :number, unique: true
```
Comment (tobiaszwaszak) [inline db/migrate/20220912190640_add_index_to_halls_number.rb]: it could be in one migration

File: app/models/hall.rb (hunk 1 above)
Comment (MonikaLukowska) [reply app/models/hall.rb]: good point, I will change it to string

**Pull request merged**